core/src/main/java/GameObjects/GameSettings.java

```diff
@@ -0,0 +1,56 @@
+package GameObjects;
+
+import Scenes.Lobby;
+
+/**
+ * The GameSettings class will store the information for the preferences that were selected in the
+ * lobby screen and will have methods that return values corresponding to the values needed in the
+ * buildUrl method of the UrlBuilderImp class.
+ */
+public class GameSettings {
+  private final String amount;
+  private final String category;
+  private final String difficulty;
+
+  public GameSettings(String amount, String category, String difficulty) {
+    this.amount = amount;
+    this.category = category;
+    this.difficulty = difficulty;
+  }
+
+  public String getCategoryValue() {
+    if (category.compareTo(Lobby.GENERAL_TEXT) == 0) {
+      return "9";
+    } else if (category.compareTo(Lobby.GEOGRAPHY_TEXT) == 0) {
+      return "22";
+    } else if (category.compareTo(Lobby.MUSIC_TEXT) == 0) {
+      return "12";
+    } else if (category.compareTo(Lobby.VIDEOGAMES_TEXT) == 0) {
+      return "15";
+    } else if (category.compareTo(Lobby.SCIENCE_NATURE_TEXT) == 0) {
```

Comment:
I disagree. It doesn't seem natural to move all the Strings from the Lobby to the GameSettings class. If we want to change the text of the buttons for Lobby we would have to go to the GameSettings class to do so.